src/lab37/exercise2/index.js

```diff
@@ -0,0 +1,20 @@
+function getFastestCar(cars) {
+  let fastestCar = { speed: 0 };
+  for (let index = 0; index < cars.length; index++) {
+    const car = cars[index];
+    if (car.speed > fastestCar.speed) {
+      fastestCar = car;
+    }
+  }
+
+  return fastestCar;
+}
```

Comment:
That is a good solution. Arrays have a few built-in methods that fit situations like this better, like `sort` that you can use to sort the cars array by speed, like this:
```js
const winner = cars.sort((car1, car2) => car2.speed - car1.speed)[0];
```